Stay aware/Sources/Scenes/View/DetailView.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  DetailView.swift
+//  Stay aware
+//
+//  Created by Олег Стригунов on 17.05.2023.
+//
+
+import UIKit
+
+class DetailView: UIView {
+    
+    //MARK: - Outlets
+    // Outlets textField
```

Comment:
```suggestion
    // MARK: - Elements
    // Outlets textField
```
Аутлеты - когда ты связываешь сториборд и файл
Тут ты верстаешь кодом, никаких аутлетов